newsletter-may-2021.rst

```diff
@@ -0,0 +1,80 @@
+.. post:: May 5, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features from May.
+
+Read the Docs newsletter - May 2021
+===================================
+
+Welcome to a new edition of our monthly newsletter, in which we would like to
+openly share with you the most relevant updates of Read the Docs,
+offer a summary of what new features we shipped to our users
+during the previous month,
+and share what things we will be focusing on in the near future.
+
+Company highlights
+------------------
+
+-  We reworked our deploy procedure to almost remove the need to interrupt builds.
+   Therefore, you should see less build retries during our deploy timeframe.
+-  After some careful deliberation,
+   we started the process to `deprecate recommonmark in favor of
+   MyST <https://github.com/readthedocs/recommonmark/issues/221>`_,
+   a better maintained alternative.
+   We are excited to see that
+   some projects have already migrated without effort,
+   and we are looking forward to helping MyST-Parser thrive!
+   Thanks to the Executable Books Project folks for creating this project.
+
```

Comment:
Should we mention our frontend position still being open here?

Reply:
Good catch!

Reply:
And our new hire?